specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/networkWatcherConnectionMonitorV1.json

```diff
@@ -0,0 +1,791 @@
+{
```

Comment:
Should the swagger have "V1" in the name?

Reply:
Yeah, there will be incompatible Connection Monitor V2 coming soon, so added V1 to differentiate

Reply:
See https://github.com/Azure/azure-rest-api-specs/pull/7075

Reply:
See https://github.com/Azure/azure-rest-api-specs/pull/7075

Reply:
OK, but would the ever be in the same version folder?

Reply:
No

Reply:
That's why I thought it wouldn't need the version number in the name, since you'd just overwrite it with the V2 content in whatever version cut over

Reply:
Connection monitor V2 will be added in networkWatcher.json and V1 will be removed, that's a bit different. We also want to explicitly state that the current CM spec is V1 since it's file will be reused until we introduce V2.

```
- Microsoft.Network/stable/2019-07-01/networkWatcher.json
- Microsoft.Network/stable/2019-06-01/networkWatcherConnectionMonitorV1.json
```
makes a bit more sense (in my opinion) than 
```
- Microsoft.Network/stable/2019-07-01/networkWatcher.json
- Microsoft.Network/stable/2019-06-01/networkWatcherConnectionMonitor.json
```